tests/test_C0_data_processing.py

```diff
@@ -1072,7 +1071,57 @@ def test_change_sign_of_feedin_tariff_zero(caplog):
         caplog.text == ""
     ), f"A msg is logged although the feed-in tariff is not changed."
 
+"""
+def test_compute_timeseries_properties(TIMESERIES_PEAK):
+    expected=
+    actual=compute_timeseries_properties(TIMESERIES_PEAK)
+    message="Expected return value:{0}, Actual return value: {1}".format(expected,actual)
+    assert actual== pytest.approx(expected),message
+
+def test_compute_timeseries_properties(TIMESERIES_TOTAL):
+    expected=
+    actual=compute_timeseries_properties(TIMESERIES_PEAK)
+    message="Expected return value:{0}, Actual return value: {1}".format(expected,actual)
+    assert actual== pytest.approx(expected),message
+
+def test_compute_timeseries_properties(TIMESERIES_AVERAGE):
+    expected=
+    actual=compute_timeseries_properties(TIMESERIES_AVERAGE)
+    message="Expected return value:{0}, Actual return value: {1}".format(expected,actual)
+    assert actual== pytest.approx(expected),message
+
+def test_compute_timeseries_properties(TIMESERIES_NORMALIZED):
+    expected=
+    actual=compute_timeseries_properties(TIMESERIES_NORMALIZED)
+    message="Expected return value:{0}, Actual return value: {1}".format(expected,actual)
+    assert actual== pytest.approx(expected),message
+"""
+
+def test_compute_timeseries_properties_TIMESERIES_in_dict_asset():
+    dict_asset= {
+        TIMESERIES: pd.Series([1,2,3]),
+        UNIT:"str",
+        LABEL: "str"
+    }
+    C0.compute_timeseries_properties(dict_asset)
+    print(dict_asset)
+
+    for parameter in [TIMESERIES_PEAK,TIMESERIES_TOTAL,TIMESERIES_AVERAGE]:
```

Comment:
To import the new variables, you need to adapt this import: https://github.com/rl-institut/multi-vector-simulator/blob/f3815e0235da676ff6181f58154d166bec67a778/tests/test_C0_data_processing.py#L9

So basically, add `TIMESERIES_AVERAGE, TIMESERIES_TOTAL` below this: https://github.com/rl-institut/multi-vector-simulator/blob/f3815e0235da676ff6181f58154d166bec67a778/tests/test_C0_data_processing.py#L79

Reply:
Understood! Thanks for your response.